src/plugins/vis_type_vislib/opensearch_dashboards.json

```diff
@@ -5,5 +5,5 @@
   "ui": true,
   "requiredPlugins": ["charts", "data", "expressions", "visualizations", "opensearchDashboardsLegacy", "visTypeVega"],
   "optionalPlugins": ["visTypeXy"],
-  "requiredBundles": ["opensearchDashboardsUtils", "visDefaultEditor"]
+  "requiredBundles": ["opensearchDashboardsUtils", "visDefaultEditor", "visAugmenter"]
```

Comment:
In the future, I still think there's a better way to set this up, so that visAugmenter can simply specify that vega-lite rendering should be used, and it wouldn't be necessary to import here. But that's something we an come back to.

Reply:
Yea that seems like a better way to structure this, but we would have to come back to that later.